exploration:
  global_frame: "world"
  base_link_frame: "/hummingbird/lidar/laser_lidar_link"
  # Exploration depth of octomap tree. 16 is best resolution, slowest exploration
  # Save frontiers on that level
  # experience shows that there is significant increase in search time by 
  # setting this parameter to < 16)
  depth: 14
  # if the candidate point is inside the radius of the previous visited points,
  # the candidate is deleted from the candidate list
  radius: 2.5
  # bounding box: necessary to limit the simulation 
  # scenario (smaller than actual gazebo scenario)
  # used for frontier detection
  bbx_minX: -38.0
  bbx_maxX: 38.0
  bbx_minY: -12.0
  bbx_maxY: 13.0
  bbx_minZ: 0.5
  bbx_maxZ: 7.0

  # Length of one side of the cube inside which we calculate information gain
  box_length: 5.0
  # Gain to scale exp function in information gain
  k_gain: 100.0
  # Constant; small lambda means that motion is cheap and gives priority
  lambda: 0.1386 


  # Total volume of the environment for exploration in m^3
  volume: 70000
sensor_model:
  # Field of view in degrees
  horizontalFOV: 360
  verticalFOV: 30
  # VelodyneVLP16 max range in meters
  max_range: 20
  pitch_angle: 10.0

  probability_hit: 0.96
  probability_miss: 0.47
  clamping_thres_min: 0.12 
  clamping_thres_max: 0.97
  
octomap:
  resolution: 0.5
  octree_depth: 16
  # Filename and file path for saving octomap
  filename: t8
  file_path: /home/manh/exploration/src/uav_frontier_exploration_3d/result/
  map_file: /home/manh/exploration/src/uav_frontier_exploration_3d/config/lv3_raw.binvox.bt
  max_waitTime: 6.75
  max_candidate: 3
  

clustering:
  # A higher bandwidth will result in fewer but larger clusters
  kernel_bandwidth: 1.5

planner:
  time_aStar: 10
